samples/trumpify/trumpify.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = os.getenv('PROJECT_ID', False)
+assert PROJECT_ID, "Missing environ variable PROJECT_ID"
+
+PROJECT_ID = int(PROJECT_ID)
+
+TW_CONSUMER_KEY = os.getenv('TW_CONSUMER_KEY', False)
+TW_CONSUMER_SECRET = os.getenv('TW_CONSUMER_SECRET', False)
+TW_ACCESS_TOKEN = os.getenv('TW_ACCESS_TOKEN', False)
+TW_ACCESS_TOKEN_SECRET = os.getenv('TW_ACCESS_TOKEN_SECRET', False)
+
+assert TW_CONSUMER_KEY, "Missing environ variable TW_CONSUMER_KEY"
+assert TW_CONSUMER_SECRET, "Missing environ variable TW_CONSUMER_SECRET"
+assert TW_ACCESS_TOKEN, "Missing environ variable TW_ACCESS_TOKEN"
+assert TW_ACCESS_TOKEN_SECRET, "Missing environ variable TW_ACCESS_TOKEN_SECRET"
+
+TWITTER_NAME = 'HillaryClinton'
```

Comment:
Is this the account we're tracking? More descriptive variable name maybe? Like INPUT_TWITTER_NAME